### PR TITLE
UAMX3-65 point all builds to UAM spec (not UTM)

### PR DESCRIPTION
**Pull request by fshihames**
UAMX3-65 point all builds to UAM spec (not UTM)
This request relates to working-in-progress code changes in four repos:
uam-apis
utm-commons
fims-deployment
nuss

Code changes are mainly for:
1. remove references of utm-apis/v4-draft at github to uam-apis/master
2. change apiVersion.txt value of v4-draft to master

I have run leafmake.sh, make.sh, itest.sh to make sure all run fine.

To be done: 
1. upload various swagger.yaml to github uam-apis/master
2. remove unnecessary repo references, e.g. uss-discovery
3. change Jenkins scripts accordingly

### CODE REVIEW

File: fims-api/swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,176 @@
+swagger: "2.0"
+info:
```
Comment (issmith1) [inline fims-api/swagger.yaml:2]: fims-api  is archaic and not part of UAM.  Do not add

File: urep-api/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+# UAS Report (UREP)
+UREP is similar in function to a Pilot Weather Report (PIREP) or Position Reports (AIREP)
```
Comment (issmith1) [inline urep-api/README.md:3]: urep should not be added, pls del

File: uss-discovery-api/swagger.yaml [hunk 3]
```diff
@@ -0,0 +1,322 @@
+swagger: '2.0'
+info:
+  description: >-
+    This API describes the RESTful interface to the USS discovery service within
+    the UTM System.  This service allows for authenticated USS operators to
```
Comment (issmith1) [inline uss-discovery-api/swagger.yaml:5]: del this file

File: vehicle/swagger.yaml [hunk 4]
```diff
@@ -0,0 +1,280 @@
+swagger: '2.0'
+info:
+  version: 'v4'
+  title: Vehicle Registration
+  description: Validates that the vehicle ID was properly registered for a NASA event.
+  contact:
```
Comment (issmith1) [inline vehicle/swagger.yaml:6]: we can keep this

**Review by issmith1 — CHANGES_REQUESTED** [review]
del files

File: fims-api/swagger.yaml (hunk 1 above)
Comment (fshihames) [reply fims-api/swagger.yaml:2]: Shall remove uam-apis/fims-api

File: urep-api/README.md (hunk 2 above)
Comment (fshihames) [reply urep-api/README.md:3]: uam-apis/urep-api is removed

File: uss-discovery-api/swagger.yaml (hunk 3 above)
Comment (fshihames) [reply uss-discovery-api/swagger.yaml:5]: uam-apis/uss-discovery-api is removed

**Pull request merged**